plugins/lookup/bitwarden.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# (c) 2022, Jonathan Lung <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: bitwarden
+    author:
+      - Jonathan Lung (@lungj) <lungj@heresjono.com>
+    requirements:
+      - bw (command line utility)
+      - be logged into bitwarden
+    short_description: Retrieve secrets from bitwarden.
+    description:
+      - Retrieve secrets from bitwarden.
```

Comment:
I think the correct capitalization is Bitwarden, with a capital B. The logo is lower-case, but in text they always seem to use a capital B. (It took me quite a while to find a sentence on their homepage which wasn't title-case or where Bitwarden wasn't at the beginning of the sentence...)
```suggestion
      - Retrieve secrets from Bitwarden.
```